### ISSUE TITLE
Decrease theme colors

### ISSUE THREAD

**Issue by MikeLockz** [ConsenSys/rimble-ui#403]
Audit the theme colors and find ones that can be removed.

**Comment by MikeLockz** [comment]
This should also get reflected in the Figma kit.

**Comment by MikeLockz** [comment]
@gesquinca - does it make sense to rename to abstract additional color names to something less tied to color and more tied to function?  When I want to override the color of the button text, instead of using `text` I have to set `white: "black"` which feels weird. Should there be a new `theme.colors.buttonText` value?

**Comment by gesquinca** [comment]
@MikeLockz yes it does. That's why we added these values to the theme recently. 

https://github.com/ConsenSys/rimble-ui/blob/master/src/theme.js#L7-L8
